modules/registry-viewing-logs.adoc

```diff
@@ -13,7 +13,7 @@ You can view the logs for the registry by using the `oc logs` command.
 for the container image registry:
 +
 ----
-$ oc logs dc/podman-registry
+$ oc logs podman-registry
```

Comment:
It is better to change the pod name to the one like this 'image-registry-79fb4469f6-llrln'